opal/mca/btl/usnic/btl_usnic_component.c

```diff
@@ -618,10 +623,11 @@ static mca_btl_base_module_t** usnic_component_init(int* num_btl_modules,
     /* Currently refuse to run if MPI_THREAD_MULTIPLE is enabled */
     if (want_mpi_threads && !mca_btl_base_thread_multiple_override) {
         opal_output_verbose(5, USNIC_OUT,
-                            "btl:usnic: MPI_THREAD_MULTIPLE not supported; skipping this component");
-        return NULL;
+                            "btl:usnic: MPI_THREAD_MULTIPLE support is in testing phase.");
```

Comment:
We did not talk about this part, but the whole reason the `_compat` code exists in the usNIC BTL is so that we can have the same code base in both the v1.10 and v2.x series.

Since we really only care about THREAD_MULTIPLE support in 2.x, you should probably put a conditional here:

* For versions < 2.0.0, do the old behavior (return NULL and the original verbose message)
* For versions >= 2.0.0, do the new behavior